worldedit-bukkit/src/test/java/com/sk89q/util/StubServer.java

```diff
@@ -0,0 +1,854 @@
+package com.sk89q.util;
```

Comment:
Move this to `com.fastasyncworldedit`, because it's a Fawe addition. Also reformat based on our recent style change. See no asterisk imports, each method component has their own line etc. I won't comment on every change, let IJ handle the reformat, it'll pick up the `.editorconfig`.